### ISSUE TITLE
Android: too less height

### ISSUE THREAD

**Issue by supernevi** [UltraStar-Deluxe/Play#300]
Issue type: Bug report

### Actual behaviour
After I just installed the android version and wanted to test the app, I noticed this visual "bug" 

![Screenshot_20220511_202820](https://user-images.githubusercontent.com/22367297/167929440-8d80fad6-d002-4897-b6be-c8539e3be247.png)

### Details
Provide some additional information:

- UltraStar Play version: 0.5.0
- Operating System + version: Android 10 (on ZTE Axon 10 Pro) with 2340x1080 display solution

**Comment by achimmihca** [comment]
I guess this is the [safe area](https://docs.unity3d.com/ScriptReference/Screen-safeArea.html) of your device, i.e. the area without any notch or other stuff that might mess up the UI.

Thankfully, Unity has an option to not ["render outside safe area"](https://docs.unity3d.com/2019.1/Documentation/Manual/class-PlayerSettingsAndroid.html) such that the developer can ignore device peculiarities.
Given the plethora of Android devices, we are not able to develop a tailored UI for all of them.

In conclusion, the reduced UI area is on purpose.
